.dotnet/src/Generated/AudioClient.cs

```diff
@@ -3,22 +3,24 @@
 #nullable disable
 
 using System;
-using System.ClientModel;
 using System.ClientModel.Primitives;
 
 namespace OpenAI.Audio
 {
     public partial class AudioClient
     {
         private readonly Uri _endpoint;
-        private const string AuthorizationHeader = "Authorization";
-        private readonly ApiKeyCredential _keyCredential;
-        private const string AuthorizationApiKeyPrefix = "Bearer";
 
         protected AudioClient()
         {
         }
 
+        internal AudioClient(ClientPipeline pipeline, Uri endpoint)
```

Comment:
These constructors that take the `ClientPipeline` and `Uri` in all the generated clients should be suppressed in favor of the existing custom constructors that take `ClientPipeline pipeline, string model, OpenAIClientOptions options`. I think the `CodeGenSuppress` attribute is already present, but the signature appears to have changed? It appears that it used to take an `ApiKeyCredential` and now it is not being suppressed correctly?

Reply:
Fixed